client/www/pages/status.tsx

```diff
@@ -64,16 +99,16 @@ const processUptimeData = (UptimeRobotResponse: any) => {
 
   const overallUptime = {
     '24h':
-      monitors.reduce((acc: number, m: any) => acc + m.uptime_ratio['24h'], 0) /
+      monitors.reduce((acc: number, m: Monitor) => acc + m.uptime_ratio['24h'], 0) /
```

Comment:
You shouldn't need to type `m` here, because typescript should already know the type of `monitors